README.md

```diff
@@ -12,7 +12,7 @@ and push to the api and s3 bucket.
     go run main.go \
         -rabbitmq-username guest \
         -rabbitmq-password guest \
-        -lagoon-api-host http://localhost:7070/graphql \
+        -lagoon-api-host http://localhost:8888/graphql \
```

Comment:
Ofc this is just an example, but which local dev setup does the api live on port 8888?